install/kubernetes/helm/istio/values.yaml

```diff
@@ -14,6 +14,11 @@ global:
   tag: circleci-nightly
   proxy:
     image: proxy
+    resources:
+      requests:
+        cpu: 100m
+        memory: 128Mi
```

Comment:
how did you get those values ?

did you try to see how that works differently with hpa ? what happens on a 16 vCPU node for instance, will proxy eventually take say 8 vCPU, the service another 8 at which point additional pod/nodes get scheduled?

Reply:
The main purpose is to set requests for the injected proxy container because otherwise the HPA won't work. Per K8s documentation:
> Please note that if some of the pod’s containers do not have the relevant resource request set, CPU utilization for the pod will not be defined and the autoscaler will not take any action for that metric.`

The values are low arbitrary values and the behaviour depends on the requested resources of the other pod containers.

Reply:
ok thanks, and with the change, does it work? (not asking for an automated regression test, just one manual test you did that shows it works now while it didn’t before)